src/webrtc/groupCall.ts

```diff
@@ -472,6 +473,12 @@ export class GroupCall extends TypedEventEmitter<
 
         for (const call of this.calls) {
             call.localUsermediaFeed.setAudioVideoMuted(muted, null);
+
+            // Disable voice activity detection when user has muted their microphone
+            // (This can't happen when push-to-talk is enabled)
```

Comment:
How so?

Reply:
Push to talk and voice activity detection kinda together kind of don't make sense.
When you use push to talk, you explicitly want to transmit. When using voice activity, you just want to send when the you reach the threshold.

We can change this obviously, but this is how it is done everywhere else as well.

Reply:
Now when I think about it, (this is more of element call thing) but we should probably clearly indicate that it is either PPT or VAD in the settings modal, so the user would clearly know that which one is used